pkg/integrations/kafka_lag_exporter/kafka_lag_exporter.go

```diff
@@ -0,0 +1,173 @@
+package kafka_lag_exporter
```

Comment:
This probably needs an inline nolint like the other packages do, it's probably going to fail the linter.